### DISCUSSION TITLE
open relative path with xdg-open

### DISCUSSION THREAD

**Discussion by esquires** [nvim-neorg/neorg#456] [Q&A] [answered]
From the documentation [here](https://github.com/nvim-neorg/neorg/blob/main/doc/neorg.norg#L166) I can form links using absolute paths to images to be opened with xdg-open but I can't seem to figure out the syntax for opening files relative to the current file. Is that possible?

I'd like to do something like this but neither seem to work:

```
{file:./mypicture.png}[mypicture]
{file:///.mypicture.png}[mypicture]
``` 

Can you point me to the right syntax for doing relative paths so that they open in an external program? It seems that the `@path` syntax works fine (e.g. for text files): `{@ myfile.txt}[myfile]`

**Comment by mrossinek** [comment, marked as answer]
This is (unforunately) not (officially) supported by the file URI scheme: https://en.wikipedia.org/wiki/File_URI_scheme

**Reply by esquires** [reply]
Thanks for the reply. Is there any interest in adding the option of opening a relative path? I imagine a relative path could be converted into an absolute path relative to the norg file path similar to the @path specification which could then be used to open a file with xdg-open using the file uri scheme

**Reply by mrossinek** [reply]
Now that we are aware of this, we can plan something, yes. I'm afraid there are higher priority issues right now, though. I will open an issue from this discussion to keep track of this :+1:
